src/plugins/query_enhancements/server/utils/plugins.ts

```diff
@@ -141,7 +141,7 @@ export const OpenSearchEnhancements = (client: any, config: any, components: any
   });
 
   enhancements.deleteJob = createAction(client, components, {
-    endpoint: `${URI.ASYNC_QUERY}/<%=queryId%>`,
+    endpoint: `${URI.ASYNC_QUERY}`,
```

Comment:
does this change break anything?

Reply:
I don't think so, there shouldn't be anything else using this currently.